jlm/llvm/opt/reduction.cpp

```diff
@@ -98,7 +98,7 @@ enable_load_reductions(jlm::rvsdg::graph & graph)
   auto nf = LoadOperation::GetNormalForm(&graph);
   nf->set_mutable(true);
   nf->set_load_mux_reducible(true);
-  nf->set_load_store_reducible(true);
+  //  nf->set_load_store_reducible(true);
```

Comment:
Could we explicitly set them to false instead of commenting them out? Also, could we add a comment above them why they are disabled? Maybe even with a link to the issues you created?